Documentation/ceph-filesystem-crd.md

```diff
@@ -39,6 +39,7 @@ spec:
     - failureDomain: host
       replicated:
         size: 3
+  preservePools: false
```

Comment:
- How about we name this property `preservePoolsOnDelete`? Then it's more obvious that this property only applies to when it is deleted. 
- I would also set this to `true` by default.

Reply:
done. I have set the default value for this setting to true in all the production crd's